lib/coroner.js

```diff
@@ -594,6 +595,36 @@ CoronerClient.prototype.delete_objects = function(universe, project, objects, pa
   this.post("/api/delete", {}, p, null, callback);
 };
 
+/*
+ * Find a service from its name.
+ */
+CoronerClient.prototype.find_service = async function (name) {
+  /*
+   * Config comes from current.json and may be stale. get a new one
+   * to find the most recent location of the service.
+   */
+  const config = JSON.parse(await this.promise("get", "/api/config", null));
+  const serviceEntry = config.services.filter(x => x.name === name)[0];
+  if (!serviceEntry) {
+    throw new Error(`No ${ name } service is configured`);
+  }
+  const endpoint = serviceEntry.endpoint;
+  if (!endpoint) {
+    throw new Error(`Service ${ name } doesn't have an endpoint`);
+  }
+  /*
+   * Frontend gets to assume that relative urls will work because it's
+   * in a browser and pointed at the same domain. We can't, because this is
```

Comment:
do we ever expect to see `//` absolute URLs, without the scheme?

Reply:
I don't think so for the time being, and think it's perhaps best to wait for that to be a roblem before trying to resolve it.